src/commands/mod.rs

```diff
@@ -26,19 +24,18 @@ pub use whoami::whoami;
 
 /// Run the given command and return its stdout.
 pub fn run(mut command: Command, command_name: &str) -> Result<(), failure::Error> {
-    info!("Running {:?}", command);
+    log::info!("Running {:?}", command);
 
     let status = command.status()?;
 
-    if status.success() {
-        Ok(())
-    } else {
+    if !status.success() {
         failure::bail!(
-            "failed to execute `{}`: exited with {}",
-            command_name,
-            status
+            "command exited with {}\n`{}`",
```

Comment:
same comment as other PR... I liked the ordering of printouts in the old message more, actually! Feel free to keep the replace logic though.